Composer/packages/client/src/recoilModel/selectors/project.ts

```diff
@@ -378,16 +379,37 @@ export const webChatEssentialsSelector = selectorFamily<WebChatEssentials, strin
   },
 });
 
+function getBaseName(filename: string, sep?: string): string {
+  if (sep) return filename.substr(0, filename.lastIndexOf(sep));
+  return filename.substring(0, filename.lastIndexOf('.')) || filename;
+}
+
+const isEmptyFile = (files, fileId) => {
+  const luFileId = fileId.replace(/\.lu$/, '');
+  return (
+    files
+      .find(({ id }) => getBaseName(id) === luFileId)
+      ?.content.trim()
+      .replace(/^>.*$/g, '')
+      .trim() === ''
+  );
+};
+
 export const allRequiredRecognizersSelector = selector({
   key: 'allRequiredRecognizersSelector',
   get: ({ get }) => {
```

Comment:
This selector reevaluates for every change in the bot asset @benbrown . Is this GetStarted section required only on bot load first up and not required afterwards?

Reply:
Yes, it should reevaluate if they add LUIS later...

Reply:
One thing to consider is reusing the logic we have in the DebugPanel to report the error for missing LUIS key, QNA Key. This functionality looks similar